src/auslib/services/releases.py

```diff
@@ -244,6 +244,14 @@ def get_assets(name, trans):
     return assets
 
 
+def get_release_names(trans, product=None):
+    where = {}
+    if product:
+        where["product"] = product
+
+    return sorted([r["name"] for r in dbo.releases_json.select(columns=[dbo.releases_json.name], where=where)])
```

Comment:
does this intentionally not pass `transaction=trans` down?

Reply:
No :). Thank your for catching that.